packages/frontend-bo/src/pages/sejours/[demandeId].vue

```diff
@@ -165,6 +182,19 @@ const {
   credentials: "include",
 });
 
+organismeStore.fetchOrganismesNonAgrees({});
+
+const isOrganismeNonAgree = computed(() => {
+  const organisme = organismeStore.organismesNonAgrees.find(
+    (o) =>
+      o.siret === demandeStore.currentDemande.organisme?.personneMorale?.siret,
+  );
+  if (organisme) {
+    libelleWarning.value = `L'organisme ${organisme.nom} n'est plus agréé depuis le ${dayjs(organisme.dateDecision).format("DD/MM/YYYY")} pour la raison suivante : ${organisme.natureDecision}`;
```

Comment:
C'est déconseillé de faire modifier une value (ce n'est pas fait pour ça et ça peut créer des problèmes de re render

Reply:
Mais du coup, ca serait quoi la bonne solution ? que mon computed renvoie le libelle ?
Et d'une manière générale, a quoi sert une variable ref si tu ne peux modifier sa value ?

Reply:
le probleme ce n'est pas de modifier une ref, mais de le faire dans une computed (c'est ok de le faire dans une computed quand tu mets des getters, setters, mais ce n'est pas le propos ici)

https://eslint.vuejs.org/rules/no-side-effects-in-computed-properties

dans ce cas là ce que tu peux faire c'est:

```javascript
const libelleWarning = computed(() => {
  const organisme = organismeStore.organismesNonAgrees.find(
    (o) =>
      o.siret === demandeStore.currentDemande.organisme?.personneMorale?.siret,
  );
  return organisme
    ? `L'organisme ${organisme.nom} n'est plus agréé depuis le ${dayjs(organisme.dateDecision).format("DD/MM/YYYY")} pour la raison suivante : ${organisme.natureDecision}`
    : '';
});
```

et utiliser libelleWarning dans le v-if